tests/python/relay/test_op_level5.py

```diff
@@ -671,10 +671,10 @@ def verify_roi_align(
             print("test on", target)
             intrp1 = relay.create_executor("graph", device=dev, target=target)
             op_res1 = intrp1.evaluate(func)(np_data, np_rois)
-            tvm.testing.assert_allclose(op_res1.asnumpy(), ref_res, rtol=1e-4)
+            tvm.testing.assert_allclose(op_res1.asnumpy(), ref_res, rtol=2e-4)
             intrp2 = relay.create_executor("debug", device=dev, target=target)
             op_res2 = intrp2.evaluate(func)(np_data, np_rois)
-            tvm.testing.assert_allclose(op_res2.asnumpy(), ref_res, rtol=1e-4)
+            tvm.testing.assert_allclose(op_res2.asnumpy(), ref_res, rtol=2e-4)
```

Comment:
Why did your changes to pooling change the precision of roi_align?

Reply:
I'm not sure actually. Taking a quick look at roi_align, I don't think it should be affected at all. I'm going to try to up the strictness of the test back to before and see if we have the same issues.

Reply:
Ok it passes still with the original tolerances, it was probably due to another bug that was fixed a long time ago